src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -156,7 +156,7 @@ function MdProgressCircularDirective($$rAF, $window, $mdProgressCircular, $mdUti
         path.attr('d', getSvgArc(animateTo, diameter, pathDiameter, rotation));
       } else {
         $$rAF(function animation(now) {
-          var currentTime = (now || getTimestamp()) - startTime;
+          var currentTime = $window.Math.min((now || getTimestamp()) - startTime, animationDuration);
```

Comment:
I find it easier to locate global variables that are being used if I reference then through `$window`.